net.stabyourself.nottetris2.yml

```diff
@@ -0,0 +1,101 @@
+id: net.stabyourself.nottetris2
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: /app/bin/nottetris2.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+modules:
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/glu/glu-9.0.0.json
+
+  # TODO: use luajit for x86 and x86_64
+  - name: lua
+    no-autogen: true
+    sources:
+      - type: archive
+        url: https://www.lua.org/ftp/lua-5.1.5.tar.gz
+        sha256: 2640fc56a795f29d28ef15e13c34a47e223960b0240e8cb0a82d9b0738695333
+      - type: patch
+        path: lua.patch
+
+  - name: libpng
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/sourceforge/libpng/libpng-1.6.34.tar.xz
+        sha256: 2f1e960d92ce3b3abd03d06dfec9637dfbd22febf107a536b44f7a47c60659f6
+
+  - name: libmng
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/sourceforge/libmng/libmng-2.0.3.tar.xz
+        sha256: 4a462fdd48d4bc82c1d7a21106c8a18b62f8cc0042454323058e6da0dbb57dd3
+
+  - name: physfs
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: http://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2
+        sha256: b77b9f853168d9636a44f75fca372b363106f52d789d18a2f776397bf117f2f1
+
+  - name: libdevil
```

Comment:
https://flathub.org/builds/#/builders/40/builds/356

this doesn't compile on aarch64. looks like it needs to be reconfigured in a manner similar to SDL here https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json

Reply:
Not sure if I understood you; see 299d8a9. Still I cannot execute it.